tests/jballerina-unit-test/src/test/resources/test-src/statements/arrays/array-initializer-expr.bal

```diff
@@ -1,3 +1,60 @@
+// Copyright (c) 2019, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
```

Comment:
License year 2023

Reply:
This is an old file added in 2019 without the license header. Hence I added it with the year 2019.